.github/workflows/unit-tests.yml

```diff
@@ -1,15 +1,23 @@
-# name: Swift
+name: Swift
 
-# on: [push]
+on: [push]
 
-# jobs:
-#   build:
+jobs:
+  build:
+    runs-on: macos-latest
```

Comment:
🤯 
I remember when macos was a no-go for the cloud and we had a stack of mac minis in the office with a sticky note on their UPS saying DO NOT UNPLUG

Reply:
Now it's an enclosure around a Mac mini 🤣